### DISCUSSION TITLE
Flight plan time marks

### DISCUSSION THREAD

**Discussion by Ascot252** [albar965/littlenavmap#1016] [Ideas]
Is there a way in Little Navmap to directly mark a flight plan with time progress marks? I don't see a way of doing this.
A simple drawing tool would be helpful to mark lines and to insert numbers etc directly on a flight plan.

At the moment, I take a screen shot of a flight plan route, paste into a Word document and then use Word to mark the flight plan route.
See attached image to show what I mean.
![Flight plan time marks](https://user-images.githubusercontent.com/130871530/232254817-d7b8c711-9467-47e2-8683-a51001f6615b.JPG)

**Comment by albar965** [comment]
I'll see if I can add this to the next major update (i.e. 3.0).

**Reply by albar965** [reply]
Created #1018 from this discussion
